src/java.desktop/share/classes/com/sun/media/sound/SoftSynthesizer.java

```diff
@@ -798,7 +798,7 @@ public InputStream run() {
                             } catch (final FileNotFoundException ignored) {
                             }
                             return null;
-                        });
+                        }).run();
```

Comment:
Let's unwrap this `RunnableAction` and put code directly in the `getDefaultSoundbank`

Reply:
No. Refactoring is explicitly a non-goal in these PRs. And minimising required refactoring is a goal.

Reply:
Will you file a separate ticket to cleanup this?
Current code looks confusing. Creating lambda just to execute it doesn't make much sense.